src/compas/scene/sceneobject.py

```diff
@@ -52,7 +55,7 @@ def draw(self):
         """The main drawing method."""
         raise NotImplementedError
 
-    @staticmethod
-    def draw_collection(collection):
-        """Drawing method for drawing an entire collection of objects."""
+    @abstractmethod
```

Comment:
should the default not be to call the pluggable and have the pluggable throw the error?